toolkit/utilitynetworks/src/main/java/com/arcgismaps/toolkit/utilitynetworks/ui/TraceColors.kt

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2024 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.arcgismaps.toolkit.utilitynetworks.ui
+
+import androidx.compose.foundation.Canvas
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.aspectRatio
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.drawscope.Stroke
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+
+/**
+ * A model for a simple color picker.
+ *
+ * @since 200.6.0
+ */
+internal object TraceColors {
+    private val colorMap = mapOf(
+        0 to Color.Green,
+        1 to Color.Blue,
+        2 to Color.Red,
+        3 to Color.Yellow,
+        4 to Color.Black
+    )
+
+    val colors: List<Color> = colorMap.values.toList()
+
+    private var chosenColor: Int = 0
+
+    /**
+     * For automatic choosing of colors for trace results
+     *
+     * @since 200.6.0
+     */
+    @Suppress("unused")
+    internal fun nextColor(): Color {
```

Comment:
Doc doesn't seem to make sense here, better to add a TODO in "Trace Options" where this should be used. 
```suggestion
    /**
     * Returns the next color in the sequence.
     *
     * @since 200.6.0
     */
    @Suppress("unused")
    internal fun nextColor(): Color {
```